libs/designsystem/src/lib/components/section-header/section-header.component.scss

```diff
@@ -0,0 +1,17 @@
+@import '~@kirbydesign/core/src/scss/utils';
+@import './section-header.utils';
+
+:host {
+  ion-item-divider {
+    --inner-padding-end: 0;
+    --color: unset;
+    --background: transparent;
+    font-weight: inherit;
+    // TODO: Determine if we really want this.
+    // This way, section header cannot be used as simple spacer
+    // Without supplying a piece of content with the needed height
```

Comment:
But maybe that is fine. I dont think we should start defining a minimum height larger than 0 here. If a simple spacing is needed, it should not be the responsibility of the section header, but rather a class provided by us, or a custom component in the project.

Reply:
Agree.